uqcsbot/scripts/meme.py

```diff
@@ -0,0 +1,192 @@
+    "fwp": "First World Problems",
+    "gandalf": "Confused Gandalf",
+    "ggg": "Good Guy Greg",
+    "grumpycat": "Grumpy Cat",
+    "hagrid": "I Should Not Have Said That",
+    "happening": "It's Happening",
+    "hipster": "Hipster Barista",
+    "icanhas": "I Can Has Cheezburger?",
+    "imsorry": "Oh, I'm Sorry, I Thought This Was America",
+    "inigo": "Inigo Montoya",
+    "interesting": "The Most Interesting Man in the World",
+    "ive": "Jony Ive Redesigns Things",
+    "iw": "Insanity Wolf",
+    "jetpack": "Nothing To Do Here",
+    "joker": "It's Simple, Kill the Batman",
+    "jw": "Probably Not a Good Idea",
+    "keanu": "Conspiracy Keanu",
+    "kermit": "But That's None of My Business",
+    "live": "Do It Live!",
+    "ll": "Laughing Lizard",
+    "mb": "Member Berries",
+    "mmm": "Minor Mistake Marvin",
+    "money": "Shut Up and Take My Money!",
+    "mordor": "One Does Not Simply Walk into Mordor",
+    "morpheus": "Matrix Morpheus",
+    "mw": "I Guarantee It",
+    "nice": "So I Got That Goin' For Me, Which is Nice",
+    "noidea": "I Have No Idea What I'm Doing",
+    "oag": "Overly Attached Girlfriend",
+    "officespace": "That Would Be Great",
+    "older": "An Older Code Sir, But It Checks Out",
+    "oprah": "Oprah You Get a Car",
+    "patrick": "Push it somewhere else Patrick",
+    "philosoraptor": "Philosoraptor",
+    "puffin": "Unpopular opinion puffin",
+    "red": "Oh, Is That What We're Going to Do Today?",
+    "regret": "I Immediately Regret This Decision!",
+    "remembers": "Pepperidge Farm Remembers",
+    "rollsafe": "Roll Safe",
+    "sad-biden": "Sad Joe Biden",
+    "sad-boehner": "Sad John Boehner",
+    "sad-bush": "Sad George Bush",
+    "sad-clinton": "Sad Bill Clinton",
+    "sad-obama": "Sad Barack Obama",
+    "sadfrog": "Sad Frog / Feels Bad Man",
+    "saltbae": "Salt Bae",
+    "sarcasticbear": "Sarcastic Bear",
+    "sb": "Scumbag Brain",
+    "scc": "Sudden Clarity Clarence",
+    "sf": "Sealed Fate",
+    "sk": "Skeptical Third World Kid",
+    "ski": "Super Cool Ski Instructor",
+    "snek": "Skeptical Snake",
+    "soa": "Seal of Approval",
+    "sohappy": "I Would Be So Happy",
+    "sohot": "So Hot Right Now",
+    "sparta": "This is Sparta!",
+    "spongebob": "Mocking Spongebob",
+    "ss": "Scumbag Steve",
+    "stew": "Baby, You've Got a Stew Going",
+    "success": "Success Kid",
+    "tenguy": "10 Guy",
+    "toohigh": "The Rent Is Too Damn High",
+    "tried": "At Least You Tried",
+    "ugandanknuck": "Ugandan Knuckles",
+    "whatyear": "What Year Is It?",
+    "winter": "Winter is coming",
+    "wonka": "Condescending Wonka",
+    "xy": "X all the Y",
+    "yallgot": "Y'all Got Any More of Them",
+    "yodawg": "Xzibit Yo Dawg",
+    "yuno": "Y U NO Guy",
+}
+
+# TODO: Would be really simple to add custom UQCS memes
+
+
+@bot.on_command("meme")
+@loading_status
+def handle_meme(command: Command):
+    """
+    !meme <meme name> "<top text>" "<bottom text>"
```

Comment:
would something like using `argparse` and `-t` from top and `-b` for bottom be better maybe? idk up to you. would remove the need for regex which is nice

Reply:
Though about that but didn't want to remove the unlikely case of wanting '-t' to be a part of the result

Reply:
surely argparse can handle that, probably just using quotations like you are now (but at least after you won't need the regex + args handling stuff)

Reply:
argparse isn't that complicated-- might just look a little messy to setup but I think it will improve the code readability a fair bit

if you implement it and decide its too complicated though i'm happy to keep what you currently have :^)

Reply:
I meant more so when it comes to using it than anything else.

Reply:
yeah also true. ultimately its up to you-- happy to go either way